packages/asc-ui/src/components/Header/examples/BodyFiller.stories.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react'
+import Heading from '../../Heading'
+import Typography from '../../Typography'
+
+const BodyFiller: React.FC = () => (
```

Comment:
IMO plz (re)move this... This isn't part of a Header, plus do we need to show other stuff besides the Header in a story? Causes much confusion I think. Stories are becoming too complex

Reply:
THanks, is just for development. Maybe moving the complexer specifi stories to a separate folder is not a bad idea